src/workflow/ArcanistPatchWorkflow.php

```diff
@@ -421,9 +435,13 @@ public function run() {
             $param);
           break;
         case self::SOURCE_DIFF:
-          if ($this->shouldUseStagingGitTags()) {
-            $this->pullBaseTagFromStagingArea($param);
-          }
+          if ($this->shouldMergeUsingStagingGitTag()) {
+            $this->mergeBranchFromStagingArea($param);
+            return 0;
+          } else
```

Comment:
shouldn't there be a brace here? for the else block.

Reply:
it's an elseif(, will fix the line break here